### DISCUSSION TITLE
Initial Boot loader for WT32-SC01 Plus, details required

### DISCUSSION THREAD

**Discussion by rarenu** [HASwitchPlate/openHASP#470] [Q&A] [answered]
Hi it is mentioned  at "https://openhasp.haswitchplate.com/0.6.3/devices/wireless-tag/" that the WT32-SC01 Plus requires flashing of initial bootloader to work as openHASP. Please share the link for the boot loader file and the associated files along with its memory locations, to make WT32-SC01 Plus as a openHASP terminal. Thank you for the help.

**Comment by fvanroie** [comment, marked as answer]
Hi,

The bootloader is included in the ***full*** firmware binary file which you can download or install from [openHASP Web installer Nightly](https://nightly.openhasp.com/).

>Important
>
>When flashing openHASP onto the ESP32 for the first time, you need to flash it over serial using the full firmware binary file. You need to write the full binary to the ESP32 flash chip at address 0x0.
>
>The full binary also contains a bootloader and partition scheme needed to properly boot the ESP32.

https://openhasp.haswitchplate.com/0.7.0/firmware/esp32/

**Comment by rarenu** [comment]
Thank you for your reply. I appreciate your prompt response.
Thank you so much.

**Reply by stovedoctor** [reply]
You have to ground gpio0

**Comment by rarenu** [comment]
Ok noted, thanks.

**Comment by rarenu** [comment]
Hi, everything works fine with my WT32-SC01-Plus display.
Thank you all for your dedication, smart work and time.

**Reply by fvanroie** [reply]
Thanks, enjoy the WT32-SC01 Plus. It's a great device.

**Comment by svenstolte** [comment]
Ich bekomme es überhaupt nicht hin, in Home Assistent sagt es mir immer die Konfiguration kann nicht geladen werden.
